docs/_deployments/aws-lambda.md

```diff
@@ -113,9 +113,11 @@ By default, Bolt listens for HTTP requests. In this section, we'll customize you
 First, install the [Serverless Express](https://github.com/vendia/serverless-express) module to transform Express HTTP requests to Lambda function events:
 
 ```bash
-npm install @vendia/serverless-express
+npm install --save @vendia/serverless-express
 ```
 
+> 💡 This guide requires version `4.x.x` or later.
```

Comment:
Can you add this in the Japanese version?

> 💡 このガイドはバージョン `4.x.x` 以上を必要とします

Reply:
Updated in commit 3208d01 (I'll squash the merge).

Thanks for catching this! I need to get better at checking the Japanese docs when we do updates.

Reply:
@mwbrooks Thanks! Now you can merge this PR 👍